packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/TraceHeader.tsx

```diff
@@ -93,9 +94,7 @@ export default function TraceHeader({
           {traceID ? (
             <React.Fragment>
               <TraceName key="name" traceName={traceName} error={error} state={state} />{' '}
-              <small key="id" className="u-tx-muted ub-pr2">
```

Comment:
what was the role of `key="id"` attribute?

Reply:
How can the new component accept additional attributes? For example, in `TraceSpanView/index.tsx` we use `common/SearchableSelect.tsx` and pass an extra attribute `data-testid=`, which is not explicitly declared in `SearchableSelect` yet still compiles, and the `SearchableSelect` includes them via `{...props}`.